libs/anaconda-assistant-mcp/src/anaconda_assistant_mcp/tools_core/shared.py

```diff
@@ -8,11 +8,31 @@
 import os
 import sys
 import subprocess
-from typing import List, Optional, Tuple
-from conda.base.context import context
+import contextlib
+from typing import List, Optional, Tuple, Generator
+from conda.base.context import context, Context
 from conda.models.channel import Channel
 
 
+@contextlib.contextmanager
+def suppress_conda_output() -> Generator[None, None, None]:
```

Comment:
This is good, but would there be a reason not to use redirect_stdout and redirect_stderr from contextlib?

https://docs.python.org/3/library/contextlib.html#contextlib.redirect_stdout